Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1167,10 +1186,75 @@ WarpXOpenPMDPlot::GetMeshCompNames (int meshLevel,
     }
 
     if ( 0 == meshLevel )
-      return;
+        return;
 
     field_name += std::string("_lvl").append(std::to_string(meshLevel));
 }
+/* Find length of field name in varname and get the theta mode index, if varname = fieldName_mode_realOrImag
+ *
+ * @param varname [IN]: name of the field variable being parsed
```

Comment:
see syntax: https://www.doxygen.nl/manual/commands.html#cmdparam
```suggestion
 * @param[in] varname name of the field variable being parsed
```

Reply:
done!